README.md

```diff
@@ -23,13 +22,18 @@ The program 'when' is currently not installed. You can install it by typing:
 
 ## Install
 
-First, tap this repository:
-
-    brew tap homebrew/command-not-found
-
-Then add following line to your `.bashrc`(bash) or `.zshrc`(zsh):
+First, tap this repository: `brew tap homebrew/command-not-found`
 
+* **Bash and Zsh**: Add following line to your `.bashrc`(bash) or `.zshrc`(zsh):
```

Comment:
> Add *the* following line

Also, please add a space between `` `.bashrc` `` and `(bash)` (same thing for zsh).